### PR TITLE
add backend to proxy to chatgpt-4

### PR DESCRIPTION
**Pull request by VoX**
requires that you set a OPENAI_API_KEY env var

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,4 +36,4 @@
       "last 1 safari version"
     ]
   }
-}
+}
```
Comment (br-ndt) [inline package.json:39]: !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

File: server/routes/wordCombine.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      },
+      {
+        "role": "assistant",
+        "content": "Steam"
+      }
```
Comment (br-ndt) [inline server/routes/wordCombine.js]: ?

**Review by br-ndt — APPROVED** [review]
LGTM aside from steam lol

**Review by br-ndt — APPROVED** [review]

File: server/routes/wordCombine.js (hunk 2 above)
Comment (VoX) [reply server/routes/wordCombine.js]: fixed

File: package.json (hunk 1 above)
Comment (VoX) [reply package.json:39]: idk probably just windows things

**Pull request merged**